docs/modules/ROOT/pages/documentation/microprofile/config/ldap.adoc

```diff
@@ -0,0 +1,154 @@
+# LDAP Config Source
+
+The LDAP config source reads configuration properties from the LDAP directory. 
```

Comment:
This statement is a bit lacking. Better to state: "The LDAP configuration source reads properties from a predefined LDAP directory"